src/components/Admin/PendingCarbonCredits.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import './PendingCarbonCredit.css';
+import PendingCarbons from './PendingCarbons';
+
+function PendingCarbonCredits({loggedInUserId}) {
+
+  const [pendingCard, setPendingCard] = useState([]);
+
+  useEffect(() => {
+    fetch(`/api/v1/carbon_credits/${loggedInUserId}`, {
+      method: "GET",
```

Comment:
-Fetch from "api/v1/carbon_credits as we want to have all carbon credits

Reply:
corrected the errors